compositor_pipeline/src/pipeline/output/whip/init_peer_connection.rs

```
@@ -0,0 +1,121 @@
+    )?;
+    let mut registry = Registry::new();
+    registry = register_default_interceptors(registry, &mut m)?;
+    let api = APIBuilder::new()
+        .with_media_engine(m)
+        .with_interceptor_registry(registry)
+        .build();
+
+    let mut stun_servers_urls = vec!["stun:stun.l.google.com:19302".to_owned()];
+
+    match env::var(STUN_SERVER_ENV) {
+        Ok(var) => {
+            if var.is_empty() {
+                info!("Empty LIVE_COMPOSITOR_STUN_SERVERS environment variable, using default");
+            } else {
+                let env_url_list: Vec<String> = var.split(',').map(String::from).collect();
+                stun_servers_urls.extend(env_url_list);
+                info!("Using custom stun servers defined in LIVE_COMPOSITOR_STUN_SERVERS environment variable");
+            }
+        }
+        Err(err) => match err {
+            VarError::NotPresent => info!("No stun servers provided, using default"),
+            VarError::NotUnicode(_) => warn!("Invalid LIVE_COMPOSITOR_STUN_SERVERS environment variable, it is not a valid Unicode, using default")
+        },
+    }
+
+    let config = RTCConfiguration {
+        ice_servers: vec![RTCIceServer {
+            urls: stun_servers_urls,
+            ..Default::default()
+        }],
+        ..Default::default()
+    };
+    let peer_connection = Arc::new(api.new_peer_connection(config).await?);
+    let video_track = Arc::new(TrackLocalStaticRTP::new(
+        RTCRtpCodecCapability {
+            mime_type: MIME_TYPE_H264.to_owned(),
+            ..Default::default()
+        },
+        "video".to_owned(),
+        "webrtc-rs".to_owned(),
+    ));
+    let audio_track = Arc::new(TrackLocalStaticRTP::new(
+        RTCRtpCodecCapability {
+            mime_type: MIME_TYPE_OPUS.to_owned(),
+            ..Default::default()
+        },
+        "audio".to_owned(),
+        "webrtc-rs".to_owned(),
+    ));
+    let _ = peer_connection.add_track(video_track.clone()).await;
```

I assume you are ignoring error here? Is it safe?